astropy/analytic_functions/tests/test_blackbody.py

```diff
@@ -32,7 +32,7 @@ def test_blackbody_scipy():
 
     """
     flux_unit = u.Watt / (u.m ** 2 * u.um)
-    wave = np.logspace(0, 8, 1e6) * u.AA
+    wave = np.logspace(0, 8, 100000) * u.AA
```

Comment:
Yeah, I see what you mean. Looking at the history, this test was originally written by @skendrew. Perhaps she could clarify, but I don't see a problem with this. I think we can even rewrite `wave` to be more sparse except around the peak, but since this is already merged, I'll just let this be unless performance is still a problem. Thanks again!